src/ToggleSwitch.js

```diff
@@ -38,12 +39,13 @@ export default class ToggleSwitch extends React.Component<ToggleSwitchProps, {}>
         );
         return (
             <div
+                aria-disabled={this.props.disabled}
                 id={this.props.id}
                 className={classes}
                 role='switch'
                 aria-label={this.props.ariaLabel}
                 aria-checked={this.props.value}
-                tabIndex='0'
+                tabIndex={this.props.disabled ? -1 : 0}
```

Comment:
As discussed on Matrix, revert this change and leave disabled ToggleSwitches in the tab order with `tabIndex='0'`

Reply:
Let's put any changes regarding tab order on hold until we resolve what to do in general about disabled controls.